deps/build.jl

```diff
@@ -123,16 +124,25 @@ catch
             end
             python = abspath(Conda.PYTHONDIR, "python.exe")
         else
-            jk_path = readchomp(`where.exe $jupyter-kernelspec`)
-            jn_path = readchomp(`where.exe $jupyter-notebook`)
-            # jupyter-kernelspec should start with "#!/path/to/python":
-            python = strip(chomp(open(readline, jk_path, "r"))[3:end])
-            # strip quotes, if any
-            if python[1] == python[end] == '"'
-                python = python[2:end-1]
+            jks_exe = joinpath(jupyter_dir, "jupyter-kernelspec.exe")
+            if !isfile(jks_exe)
+                jk_path = readchomp(`where.exe $jupyter-kernelspec`)
+                jn_path = readchomp(`where.exe $jupyter-notebook`)
+                # jupyter-kernelspec should start with "#!/path/to/python":
+                python = strip(chomp(open(readline, jk_path, "r"))[3:end])
+                # strip quotes, if any
+                if python[1] == python[end] == '"'
+                    python = python[2:end-1]
+                end
+            elseif isfile(jn_path) || error("$jn_path not found")
```

Comment:
This isn't right, because you have to check `isfile(jn_path)` *after* assigning it.